core/server/common/src/main/java/alluxio/cli/ValidateEnv.java

```diff
@@ -79,6 +97,14 @@
       "proxy.web.port.available",
       new PortAvailabilityValidationTask(ServiceType.PROXY_WEB, ALLUXIO_PROXY_CLASS));
 
+  // security configuration validations
+  private static final ValidationTask MASTER_SECURE_HDFS_VALIDATION_TASK = registerTask(
+      "master.ufs.hdfs.security.kerberos",
+      new SecureHdfsValidationTask("master"));
```

Comment:
nit: define the keys for `PRINCIPAL_MAP` in `SecureHdfsValidationTask` as a constant

Reply:
Done.